carbon-design-system/carbon
# test(containedlist): move to stable - add vrt, avt, storybook docs

Closes #12319

Final steps to mark ContainedList as stable

#### Changelog

**New**

- Component has an mdx document that follows our outline
- mdx document provides a prop table
- Component is tested via VRT for all stories
- Component is tested via AVT for the initial render state
- Added platform metadata `carbon.yml`, added live demo .mdx

**Changed**

- Move story file out of `/next`
- Move story out from under `Experimental` to `Components` section
- Hide `action`, `children`, `className` from all stories' controls pane

**Removed**

- `unstable_` prefix from ContainedList

#### Testing / Reviewing

- View all stories, ensure they render and function correctly
- Make sure exports look correct - `unstable_` prefix should be gone from both ContainedList and ContainedListItem
